src/buildApplication/generateFunctionsMap.ts

```diff
@@ -354,6 +362,27 @@ async function buildWebpackChunkFiles(
 	}
 }
 
+/**
+ * Next.js generates a favicon nodejs function instead of providing the favicon as
+ * a standard static asset. Since it is not using the edge runtime we can't use the
+ * function itself, so this function tries to see if it can find the standard favicon
+ * in the ./static/media directory generated by Next.js and moves that one in the static
+ * output directory instead
+ *
+ * (Note: this is hopefully just a temporary solution, hopefully Next.js will allow to opt
+ *        out from the favicon function generation or allow it to use the edge runtime)
```

Comment:
- [ ] TODO: before merging create a new issue in the Next repository and reference it here

Reply:
I've removed the comment entirely as I actually don't think Next is going to do anything about this (not soon at the very least) so this fix might need to stay 😕 😓 

@jculvey does this shound ok-ish to you?